test/functional/ex_cmds/cd_spec.lua

```diff
@@ -0,0 +1,99 @@
+-- Specs for :cd, :tcd, :lcd and hastabdir(), haslocaldir()
+
+local helpers = require('test.functional.helpers')
+local execute, eq, clear, eval, feed =
+  helpers.execute, helpers.eq, helpers.clear, helpers.eval, helpers.feed
+
+
+-- These directories will be created for testing
+local directories = {
+  'Xtest-functional-ex_cmds-cd_spec.1', -- Tab
+  'Xtest-functional-ex_cmds-cd_spec.2', -- Window
+  'Xtest-functional-ex_cmds-cd_spec.3', -- New global
```

Comment:
This is good, we should get in this habit (as @ZyX-I suggested). Or provide a helper function that auto-generates a name based on the test file.